tests/test_interface.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo "Running cc-snapshot interface tests..."
+
+fail() {
+  echo "FAIL: $1"
+  exit 1
+}
+
+pass() {
+  echo "PASS: $1"
+}
+
+CC_SNAPSHOT=../cc-snapshot
+
+# Test 1: Help option (-h)
+output=$(sudo "$CC_SNAPSHOT" --dummy -h 2>&1) || true
```

Comment:
Let's make 1 tweak to the `cc-snapshot` script so we don't have to use sudo for some of our testing.

We can add `&& -z ${TESTING_SKIP_ROOT_CHECK}` here:
https://github.com/ChameleonCloud/cc-snapshot/blob/master/cc-snapshot#L3

E.g.
```
if [[ $EUID -ne 0 && -z ${TESTING_SKIP_ROOT_CHECK} ]]; then
```

and then when we call `cc-snapshot` in your script it can be called like:
`TESTING_SKIP_ROOT_CHECK=1 ${CC_SNAPSHOT} -h`

Also, is the `--dummy` flag here serving a purpose yet since it hasn't been added? Should this test just check `-h`?